base/inference.jl

```diff
@@ -3365,40 +3460,44 @@ function optimize(me::InferenceState)
     type_annotate!(me)
 
     # run optimization passes on fulltree
-    force_noinline = false
+    force_noinline = true
```

Comment:
If you're changing the meaning of this to the complete opposite, can you please change the name of the variable?